### DISCUSSION TITLE
Razzle help

### DISCUSSION THREAD

**Discussion by octavz** [tailwindlabs/tailwindcss#1688] [General]
Hi, 
It is unclear how to use tailwind together with Razzle.
Can you add to the home page or some kind of best practices on how these two should work together?

Thank You,

**Comment by adamwathan** [comment]
I’d start with this plugin and see if it helps:

https://github.com/rhodee/razzle-plugin-postcss/blob/master/README.md

As long as you can setup postcss plugins you can get Tailwind working 👍🏻

**Comment by octavz** [comment]
I am a newbie, Razzle is basically a no config setup, one can run [almost] any mainstream or non mainstream js-framework with basically no extra configuration. 

I find the learning curve for Tailwind disappointing. Honestly it took me less to learn how to use React.js compared to the time already spent on learning/tooling/configuration for this framework.

Anyway thank you for your time, my project is small I probably don't need it.
